css/liveblog.css

```diff
@@ -150,8 +150,8 @@ a.liveblog-form-entry::-webkit-input-placeholder {
 	margin: 1px;
 	border: solid 1px rgba(0, 0, 0, 0.0);
 	padding: 2px 1px;
-	width: 20px;
-	height: 20px;
+	width: 24px;
```

Comment:
I think the icons are supposed to squares. Is there a reason the width and height are different?